server/server.js

```diff
@@ -142,45 +142,49 @@ Server.prototype._handleSocketMessage = function(socket, data) {
     logging.warn('#socket.message - rejected', socket.id, data);
     return;
   }
+  
+  this._processMessage(socket, message);
+}
+
+Server.prototype._processMessage = function(socket, message) {
+  var messageType = this._getMessageType(message),
+      resource = this._getResource(message, messageType);
 
-  if (!this._messageAuthorize(message, socket)) {
+  if (!this._authorizeMessage(socket, message, messageType)) {
```

Comment:
I am sure this is something Pat would like to be left alone. But I realized I preferred verb first after reading through the code. @bolddane What do you think? Is this ok?